client/src/Offer.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useParams } from "react-router-dom";
+import Paper from '@material-ui/core/Paper';
+import InputBase from '@material-ui/core/InputBase';
+import { NavLink } from 'react-router-dom';
+import SearchIcon from '@material-ui/icons/Search';
+import Button from '@material-ui/core/Button';
+import ArrowBackRoundedIcon from '@material-ui/icons/ArrowBackRounded';
+import Typography from '@material-ui/core/Typography';
+import TodayOutlined from '@material-ui/icons/TodayOutlined';
+import RoomOutlined from '@material-ui/icons/RoomOutlined';
+
+import { dumbyData } from './assets/temp';
+
+import type { Theme } from '@material-ui/core/styles';
+import type { Asset as AssetT } from './types';
+
+
+const useStyles = makeStyles((theme: Theme) => ({
+    topBar: {
+        backgroundColor: 'white',
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        padding: '10px 5%',
+        borderRadius: 0,
+        alignItems: 'center',
+    },
+    searchInput: {
+        display: 'flex',
+        alignItems: 'center',
+        paddingLeft: '10px',
+    },
+    // TODO i'm just copying these from other files,
+    // either make them a component or make the styles shared objects somewhere
+    iconButton: {
+        padding: 10,
+        '&:hover': {
+            backgroundColor: 'inherit',
+            borderRadius: '10px',
```

Comment:
i'm just copying a lot of these style objects from other files, we should discuss a better long term solution
just lots of little components?
idk really know material-ui theming but maybe something in there?

Reply:
I think there probably is a better way, but in the past I have saved styles that I found myself reusing all the time into a separate file and imported them into the components, or created styled components. I bet there is a way you can save styles directly to the Theme.